README.md

```diff
@@ -5,7 +5,7 @@ argument used will be ```--relaxng``` instead of ```--schema```. Otherwise
 
 -----------------------------------------------------------------------------------
 
-Online demo at http://syssgx.github.com/xml.js/
+Online demo at [http://syssgx.github.com/xml.js/](https://syssgx.github.io/xml.js/)
```

Comment:
```suggestion
Online demo at https://syssgx.github.io/xml.js/
```
(otherwise the visible link is the old, wrong one?

Reply:
Well, that's a mistake. I've fixed it.